models/fixtures/action.yml

```diff
@@ -17,8 +17,8 @@
   act_user_name: user3
   repo_id: 3
   repo_user_name: user3
-  repo_name: repo3 # TODO old or new name?
-  is_private: false
+  repo_name: repo3
+  is_private: true
```

Comment:
The repo with id 3 is private, so this action should also be marked as private. I'll update the PR description to include this.